cxx-checks/src/test/resources/checks/UsingNamespaceInHeader.h

```diff
@@ -8,7 +8,7 @@ namespace Foo
     void fooFunc( );
 }
 
-using namespace std;
+using namespace std; // error
```

Comment:
just for your information: this error is equivalent to clang-tidy's rule `google-global-names-in-headers` aka "using declarations in the global namespace in headers are prohibited"